test/stat_tracker_test.rb

```diff
@@ -0,0 +1,42 @@
+require './test/test_helper'
+require './lib/stat_tracker'
+
+class StatTrackerTest < Minitest::Test
```

Comment:
StatTracker test doesn't seem to be working. Need to revisit

Reply:
Pushed changes